Timer.php

```diff
@@ -81,9 +100,10 @@ public static function signalHandle()
      * @param callable $func
      * @param mixed    $args
      * @param bool     $persistent
-     * @return int|false
+     * @param int      $persistent_timer_id
+     * @return int|bool
      */
-    public static function add($time_interval, $func, $args = array(), $persistent = true)
+    public static function add($time_interval, $func, $args = array(), $persistent = true, $persistent_timer_id = 0)
```

Comment:
Thank you for your post. It is a good pr.
But `Timer::add` has a lot of parameters. I don't think we should add more parameters to it which will make the Timer more complicated to use. I would appreciate it if the parameter `persistent_timer_id` can be removed.

Reply:
ok, i will find out whether it has a better workaround.